gdk4-wayland/Cargo.toml

```diff
@@ -28,7 +28,8 @@ gdk = {path = "../gdk4", package = "gdk4", version = "0.5.0"}
 gio = {git = "https://github.com/gtk-rs/gtk-rs-core", version = "0.16", features = ["v2_66"]}
 glib = {git = "https://github.com/gtk-rs/gtk-rs-core", version = "0.16", features = ["v2_66"]}
 libc = "0.2"
-wayland-client = {version = "0.29", features = ["use_system_lib"], optional = true}
+wayland-client = {version = "0.30.0-beta.3", optional = true}
+wayland-backend = {version = "0.1.0-beta.3", optional = true, features = ["client_system"]}
```

Comment:
Do we need the wayland-backend crate or can it be enabled as a feature of wayland-client directly? as I don't see it used directly from wayland_backend.

Reply:
Compilation fails without it...

Reply:
The client_system feature can only be enabled via the wayland-backend crate FYI.